hypertrace-view-generator/hypertrace-view-generator/src/test/java/org/hypertrace/viewgenerator/generators/BackendEntityViewGeneratorTest.java

```diff
@@ -0,0 +1,64 @@
+package org.hypertrace.viewgenerator.generators;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.avro.file.DataFileReader;
+import org.apache.avro.specific.SpecificDatumReader;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.StructuredTrace;
+import org.hypertrace.viewgenerator.api.BackendEntityView;
+import org.junit.jupiter.api.Test;
+
+public class BackendEntityViewGeneratorTest {
+
+  @Test
+  public void testBackendEntityViewGenerator_HotrodTrace() throws IOException {
+    URL resource =
```

Comment:
Quite a few test are reading trace, can you abstract out this method as  a static utility?

Reply:
done